util-core/src/test/scala/com/twitter/concurrent/AsyncStreamTest.scala

```diff
@@ -426,4 +426,12 @@ private object AsyncStreamTest {
   def undefined[A]: A = throw new Exception
 
   def toSeq[A](s: AsyncStream[A]): Seq[A] = await(s.toSeq())
+
+  def fromSeq[A](s: Seq[A]): AsyncStream[A] =
```

Comment:
Done. Github isn't collapsing this part somehow, maybe because the diff didn't change where you marked it.